src/main/java/io/split/android/client/ServiceEndpoints.java

```diff
@@ -0,0 +1,123 @@
+package io.split.android.client;
+
+import androidx.annotation.NonNull;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public class ServiceEndpoints {
+
+    private static final String API_ENDPOINT = "https://sdk.split.io/api";
+    private static final String EVENTS_ENDPOINT = "https://events.split.io/api";
+    private static final String SSE_AUTH_SERVICE_ENDPOINT = "https://auth.split-stage.io/api";
+    private static final String STREAMING_SERVICE_ENDPOINT = "https://realtime.ably.io/sse";
+
+    private String mApiEndpoint = API_ENDPOINT;
+    private String mEventsEndpoint = EVENTS_ENDPOINT;
+    private String mSseAuthServiceEndpoint = SSE_AUTH_SERVICE_ENDPOINT;
+    private String mStreamingServiceEndpoint = STREAMING_SERVICE_ENDPOINT;
+
+    private ServiceEndpoints() {
+    }
+
+    public String getApiEndpoint() {
+        return mApiEndpoint;
+    }
+
+    private void setApiEndpoint(String endpoint) {
+        this.mApiEndpoint = endpoint;
+    }
+
+    public String getEventsEndpoint() {
+        return mEventsEndpoint;
+    }
+
+    private void setEventsEndpoint(String endpoint) {
+        this.mEventsEndpoint = endpoint;
+    }
+
+    public String getAuthServiceEndpoint() {
+        return mSseAuthServiceEndpoint;
```

Comment:
just a comment, but for consistency, you could rename `mSseAuthServiceEndpoint` to `mAuthServiceEndpoint` or `getAuthServiceEndpoint` to `getSseAuthServiceEndpoint`